src/rpc/mempool.cpp

```diff
@@ -42,6 +42,7 @@ static RPCHelpMan sendrawtransaction()
             {"maxfeerate", RPCArg::Type::AMOUNT, RPCArg::Default{FormatMoney(DEFAULT_MAX_RAW_TX_FEE_RATE.GetFeePerK())},
              "Reject transactions whose fee rate is higher than the specified value, expressed in " + CURRENCY_UNIT +
                  "/kvB.\nSet to 0 to accept any fee rate.\n"},
+            {"no_burning", RPCArg::Type::BOOL, RPCArg::Default{true}, "If true, reject transactions with non-zero OP_RETURN outputs, which are unspendable and a 'burning' of coins."},
```

Comment:
I wonder if this can indicate the maximum value for a data output (default: 0) instead of a true/false

Reply:
marco wheres your responsiblelity of your own words that you said in our chat by telegram?
you already receive 0.15 btc from me

Reply:
I think`max_burn_amount` is the right name here, similar to `maxfeerate`, since that's what this check is most similar to.

Reply:
Thanks for the suggestion, I have changed the parameter to `maxburnamount` (I avoided snake case in the json parameter to keep consistent with `maxfeerate` and used snake case for our c++ variable `max_burn_amount`) and ensure that our `nValue` is not greater.